baselines/Google Chat Minimum Viable Secure Configuration Baseline v0.1.md

```diff
@@ -300,3 +302,52 @@ Agencies SHOULD configure DLP rules to block or warn on sharing files with sensi
 9.  Select the appropriate action to warn or block sharing, based upon the agency's individual requirements.
 10. In the **Alerting** section, choose a severity level, and optionally, check **Send to alert center to trigger notifications**.
 11. Review the rule details, mark the rule as **Active**, and click **Create.**
+
+## 7. Content Reporting
+
+This section covers the content reporting functionality, a feature that allows users to report messages that violate organizational guidelines to workspace admins.
+
+### Policies
+
+#### GWS.CHAT.7.1v0.1
+Chat content reporting SHALL be enabled for all conversation types.
+
+- Rationale
+  - Chat messages could potentially be used as an avenue for phishing, malware distribution, or other security risks. Enabling this feature allows users to report any suspicious messages to workspace admins, increasing threat awareness and facilitating threat mitigation. By selecting all conversation types, agencies ensure that their users are able to report risky messages regardless of the conversation type.
+- Last Modified: February 13, 2024
+
+- MITRE ATT&CK TTP Mapping
+  - [T1530: Data from Cloud Storage](https://attack.mitre.org/techniques/T1530/)
+
+#### GWS.CHAT.7.2v0.1
+All reporting message categories SHOULD be selected.
+
+- Rationale
+  - Users may be uncertain what kind of messages should be reported. Enabling all message categories can help users infer which types of messages should be reported.
+- Last Modified: February 13, 2024
+
+- MITRE ATT&CK TTP Mapping
+  - [T1530: Data from Cloud Storage](https://attack.mitre.org/techniques/T1530/)
+
+### Resources
+- [Set-up content moderation for Chat](https://support.google.com/a/answer/13471510?hl=en)
+
+### Prerequisites
+-   Chat history must be enabled to configure settings in this group.
```

Comment:
It's not that history needs to be enabled for changing these settings, rather it needs to be enabled for users to report messages.
```suggestion
-   Chat history must be enabled for users to be able to report messages.
```